src/components/include/utils/threads/async_runner.h

```diff
@@ -111,6 +113,11 @@ class AsyncRunner {
      */
     void waitForDelegate();
 
+    /**
+     * @bref clearDelegateQueue delete leftover delegates in the queue
+     */
+    void clearDelegateQueue();
```

Comment:
@SNiukalov Wrong naming(Coding style)

Reply:
@AByzhynar 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/41/commits/dff6e3e9b9309028a4371c081b7dc6216bf21bcc